examples/eager_mode.py

```diff
@@ -0,0 +1,17 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+import torch
+
+from torch_mlir.eager_mode.torch_mlir_tensor import TorchMLIRTensor
+
+a = TorchMLIRTensor(torch.randn(5, requires_grad=True))
+b = TorchMLIRTensor(torch.randn(5, requires_grad=True))
+
+c = a + b
+d = a * b
+e = c / d
+
+e.sum().backward()
```

Comment:
print the sum.
Also print out some gradients after the backward call too to show off how seamless this all is :)

(also, random question: does TorchMLIRTensor also cause the backward graph to execute through Torch-MLIR?)

Reply:
>also, random question: does TorchMLIRTensor also cause the backward graph to execute through Torch-MLIR

yup - it's like magic (the `grad_fn` gets attached to the `TorchMLIRTensor` since it's a subclass of `torch.Tensor` and then during backwards `__torch_dispatch__` fires just like during the forward pass).